queries/rnoweb/injections.scm

```diff
@@ -0,0 +1,7 @@
+(latex) @latex @combined
+(rchunk
+	(renv_content) @r @combined
```

Comment:
`@combined` is not yet supported by nvim (or it has a different understanding of combined)

Reply:
I'm having a hard time finding in the documentation where I saw the `@combined` flag, but if i exclude it from the (latex) query, then there are errors throughout the document where (rchunk) and (rinline) nodes break latex environments. It appears to be doing something useful to me. Can you point me to where in the neovim code `@combined` is defined?

Reply:
I think it should be ok. It's just not behaving the way tree-sitter does.  It is defined or better used in languagetree.lua in neovim